src/ChunkFieldValue.jsx

```diff
@@ -22,7 +22,19 @@ export function ChunkFieldValue({ children, identifier, ...props }) {
     return null;
   }
 
-  return renderChunk(fieldChunk, props);
+  let dummyFieldChunk;
+  if (chunk && chunk.placeholder) {
+    dummyFieldChunk = {...fieldChunk, 
+      identifier: "",
+      content: fieldChunk.chunk_type === 'image' ? 'https://editmode.com/upload.png' : ""
+    }
+    if (dummyFieldChunk.chunk_type === 'rich_text') {
+      props.style ? props.style['width'] = "100px" : props.style = {minWidth: "200px"}
```

Comment:
I think using a ternary operator is not advisable cause it is an expression and preferrable be used together with an assignment statement like:

```javascript
const someVar = condition ? true : false;
```

I think a simple `if-else` statement is appropriate?

Also can `props.style ? props.style['width'] = "100px" : props.style = {minWidth: "200px"}` be like:

```javascript
if (props.style) {
  props.style.width = '100px'
else {
  props.style.minWidth = '200px'
}
```

Kinda like using dot notation if possible?